### ISSUE TITLE
training loss

### ISSUE THREAD

**Issue by idejie** [ShuangLI59/Person-Search-with-Natural-Language-Description#9]
I re-implement this model with Pytorch, but the loss is around 0.69 and does not drop in the training stage.

I want to know the loss when you training.

Thanks~
